static/app/views/alerts/rules/metric/details/sidebar.tsx

```diff
@@ -139,6 +142,24 @@ function TriggerDescription({
   );
 }
 
+function getColor(theme: Theme, type: 'critical' | 'warning' | 'success') {
+  if (theme.isChonk) {
+    return type === 'critical'
+      ? (theme as DO_NOT_USE_ChonkTheme).colors.chonk.red400
+      : type === 'warning'
+        ? (theme as DO_NOT_USE_ChonkTheme).colors.chonk.yellow400
+        : (theme as DO_NOT_USE_ChonkTheme).colors.chonk.green400;
+  }
+  return type === 'critical'
+    ? theme.errorText
+    : type === 'warning'
+      ? theme.warningText
+      : theme.successText;
+}
+const StyledIconDiamond = styled(IconDiamond)<{type: 'critical' | 'warning' | 'success'}>`
+  fill: ${p => getColor(p.theme, p.type)};
+`;
+
```

Comment:
The IconDiamond is deprecated and will be removed in the future so I didnt bother making a dedicated component for it